app/database/migrations/20230428075403_add_qr_to_db/migration.sql

```diff
@@ -19,6 +19,3 @@ ALTER TABLE "QR" ADD CONSTRAINT "QR_itemId_fkey" FOREIGN KEY ("itemId") REFERENC
 
 -- AddForeignKey
 ALTER TABLE "QR" ADD CONSTRAINT "QR_userId_fkey" FOREIGN KEY ("userId") REFERENCES "User"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
-
```

Comment:
We removed this for testing purposes when we were checking the issue you had. There should be no changes in this file. Please revert them